components/docs-page.js

```diff
@@ -122,7 +122,14 @@ function DocSection(props) {
           dangerouslySetInnerHTML={{__html: mdToHTMLUnwrapped(title)}}
         />
       </Anchor>
-      <h4 dangerouslySetInnerHTML={{__html: mdToHTMLUnwrapped(subtitle)}} />
+      <glamorous.Div
```

Comment:
It's not really semantically accurate to use an `h4` and it looks not awesome. So I formatted this a little differently in a way I think is nicer